components/gallery/GalleryItemButton/GalleryItemShareBtn.vue

```diff
@@ -1,66 +1,9 @@
 <template>
```

Comment:
remove this component if empty, and replace `GalleryItemShareBtn` with `ShareButton`